transposer/plugins/org.polarsys.kitalpha.transposer.rules.handler.mapping.editor/META-INF/MANIFEST.MF

```diff
@@ -14,6 +14,7 @@ Export-Package: org.polarsys.kitalpha.transposer.rules.handler.rules.common.pres
 Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.core.resources;visibility:=reexport,
  org.polarsys.kitalpha.transposer.rules.handler.mapping.edit;visibility:=reexport,
+ org.polarsys.kitalpha.transposer.rules.handler.mapping.edit;visibility:=reexport,
```

Comment:
I do not see any modification here.

Reply:
done. The required was duplicated by the generation !